packages/next/package.json

```diff
@@ -271,6 +271,7 @@
     "ora": "4.0.4",
     "os-browserify": "0.3.0",
     "p-limit": "3.1.0",
+    "p-queue": "6.6.2",
```

Comment:
starting with `^7.0.0`, p-queue is ESM-only, which seemed like it'd be an issue? not sure

Reply:
Is there an alternative that would allow us to get bugfixes i.e. a library where we can use the latest version?

Reply:
We've had easier times using CJS for the ones we compile but technically could be ESM just need to tweak some resolving in the `taskfile` when ncc'ing it

Reply:
Also, from their readme:
> This package is native [ESM](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and no longer provides a CommonJS export. If your project uses CommonJS, you'll have to [convert to ESM](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) [...] **You can also use [version 6](https://github.com/sindresorhus/p-queue/tree/v6.6.2) instead which is pretty stable. We will backport security fixes to v6 for the foreseeable future.**

So probably should be fine?

Reply:
Figured out that i can just pass `esm: false` to ncc and it'll emit CJS, so we can use the latest version: [`9798e04` (#65950)](https://github.com/vercel/next.js/pull/65950/commits/9798e04e6ad3efb30becdc1169a4575d9ea90f1a)

Reply:
Ok, i'm gonna stay on 6.0.1 in this PR and update it later: https://github.com/vercel/next.js/pull/65987
The update is making some random tests fail because the package contains private class properties, so let's deal with that later